site/content/en/docs/faq/_index.md

```diff
@@ -39,7 +39,7 @@ minikube start --driver=hyperv
 ```
 Alternatively, you can use `minikube image build`  instead of `minikube docker-env` and `docker build`.
 
-## Can I start minikube without Kuberentes runninng?
+## Can I start minikube without Kuberentes running?
```

Comment:
can you swap Kuberentes to Kubernetes while we're here? thanks for catching this

Reply:
Definitely: 0550d97966db2b70300cc25501c9c8fc56d83548